staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -50,15 +50,14 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 		defer recordAuthMetrics(resp, ok, err, apiAuds, authenticationStart)
 		if err != nil || !ok {
 			if err != nil {
-				klog.Errorf("Unable to authenticate the request due to an error: %v", err)
+				klog.V(2).Infof("request (%s): unable to authenticate due to an error: %v", shortRequestInfo(req), err)
```

Comment:
the PR description didn't mention lowering an error to info-level... why is this being done?

Reply:
The log lever lowering was a mistake, I too think it should be kept on the same level.